api/RestfulService.php

```diff
@@ -175,7 +191,7 @@ public function request($subURL = '', $method = "GET", $data = null, $headers =
 			$method,
 			$data,
 			array_merge((array)$this->customHeaders, (array)$headers),
-			$curlOptions + (array)$this->config()->default_curl_options,
+			$curlOptions + (array)$this->get_default_curl_options(),
```

Comment:
> Easy to change though.

Would you mind?

Static methods should be called statically; dynamic methods dynamically. There's no benefit to invoking it in a different context here.

Yes, we should be using `static` instead of `self`, in general, but I can't see how that principle applies here.